src/mongo/client/wire_protocol_writer.cpp

```diff
@@ -28,8 +28,11 @@ WireProtocolWriter::WireProtocolWriter(DBClientBase* client) : _client(client) {
 void WireProtocolWriter::write(const StringData& ns,
                                const std::vector<WriteOperation*>& write_operations,
                                bool ordered,
+                               bool bypassDocumentValidation,
                                const WriteConcern* writeConcern,
                                WriteResult* writeResult) {
+    invariant(bypassDocumentValidation == false);
```

Comment:
There is no way for users to reach this if they use the `DBClient` API. If they use `WireProtocolWriter` directly, then they can use. Is `WireProtocolWriter` considered part of the public API of the driver?